lib/globalize.js

```diff
@@ -7,5 +7,10 @@
 var path = require('path');
 var SG = require('strong-globalize');
 
+if (process.env.NODE_ENV === 'test') {
+  module.exports = SG({language: 'en'});
```

Comment:
@Setogit I am wondering, what environment variables should we change in our tests in order to force strong-globalize to use `en` language when running the test suite? I'd like to avoid this custom SG initialization used by our tests only.